src/Foundatio.Extensions.Hosting/Jobs/ScheduledJobManager.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Foundatio.Caching;
+using Foundatio.Jobs;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Foundatio.Extensions.Hosting.Jobs;
+
+public interface IScheduledJobManager
+{
```

Comment:
Seems like it would be nice to know if it's running or current state... one might take this into account before you update the schedule?

Reply:
Maybe, but I'll add it when someone actually needs it.